config.toml

```diff
@@ -430,3 +430,36 @@ DefaultContentLanguage = "en"
   weight = 6
   name = "Twitter"
   url = "https://twitter.com/redox_os"
+
+[languages.pt]
+    weight = 0
```

Comment:
I have no understanding of the meaning for the weight attribute. Anyway, I see that all [languages.*] sections are sequential from English (weight=0) to Esperando (weight=12). Based on that, can I say that languages.pt should have weight=13?